server/src/internalClusterTest/java/org/opensearch/indices/mapping/UpdateMappingIntegrationIT.java

```diff
@@ -390,7 +390,7 @@ private void assertConcreteMappingsOnAll(final String index, final String... fie
     }
 
     /**
-     * Waits for the given mapping type to exists on the master node.
+     * Waits for the given mapping type to exists on the cluster-manager node.
      */
     private void assertMappingOnMaster(final String index, final String... fieldNames) {
```

Comment:
`assertMappingOnMaster` -> `assertMappingOnClusterManager`?

Reply:
Changed the method name in the new commit https://github.com/opensearch-project/OpenSearch/pull/2521/commits/87d07e5b81ed67d4f72842ef335ae627839e4954. Thank you for catching it!